src/pet.js

```diff
@@ -0,0 +1,71 @@
+const MAX_FITNESS_LEVEL = 10;
+const MIN_HUNGER_LEVEL = 0;
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = 10;
+    this.children=[];
+}
+
+Pet.prototype = {
+    get isAlive() {
+        return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+}
+
+Pet.prototype.growUp = function () {
+    if(!this.isAlive){
+        throw new Error('Your pet is no longer alive :(');
+    }
+    this.age += 1;
+    this.hunger += 5;
+    this.fitness -= 3;
```

Comment:
This would be a perfect example of when to use some of those example constants I put in the comment above (`AGE_INCREMENT`, `FITNESS_INCREMENT`, `HUNGER_INCREMENT`)